kore/src/Kore/JsonRpc.hs

```diff
@@ -298,6 +362,12 @@ respond
       where
         couldNotVerify err = ErrorObj "Could not verify KORE pattern" (-32002) err
 
+        serverError detail payload =
```

Comment:
Should the errors all have different codes? Per the JSON RPC spec, you should use -32000 to -32099 for user errors. Perhaps make all errors a big enum and then use [`fromEnum`](https://hackage.haskell.org/package/base/docs/Prelude.html#v:fromEnum) to assign the code?

Reply:
Good idea, but more cumbersome to keep the doc.s in sync.
For now I have arbitrarily used `-32032` (leaving some room for more user errors before this server error), and added to the documentation.